MAUI/DataGrid/grid-events.md

```diff
@@ -38,6 +38,20 @@ private void dataGrid_CellTapped(object sender, DataGridCellTappedEventArgs e)
 {% endhighlight %}
 {% endtabs %}
 
+### CellTappedCommand and its Parameter
+When we tapped the cell, the `CellTappedCommand` binds to the command which you defined in your ViewModel. With this we can also use the `CellTappedCommandParameter` to be passed to the `Command` when it is executed.
```

Comment:
The command will be invoked while tapping a cell in the SfDataGrid. The instance of `DataGridCellTappedEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellTappedCommandParameter` property.

Reply:
Changed